src/app/routes/utils/regex/utils/index.js

```diff
@@ -35,11 +30,17 @@ const getWorldServices = services => {
     'cymrufyw',
     'naidheachdan',
     'archive',
+    'scotland',
   ];
 
   return services.filter(service => !publicServices.includes(service));
 };
 
+export const getArticleSwRegex = services => {
+  const serviceRegex = getServiceRegex(getWorldServices(services));
+  return `/:service(${serviceRegex})/:local(${articleLocalRegex})/sw.js`;
```

Comment:
Given that the `articleLocalRegex` was for the public services, does it now make sense to remove this, and hardcode `articles` instead...?

```suggestion
  return `/:service(${serviceRegex})/articles/sw.js`;
```

Reply:
Good idea. Updated